sites/fast-creator/app/creator.tsx

```diff
@@ -334,6 +334,16 @@ class Creator extends Editor<{}, CreatorState> {
                 });
                 updatedState.previewReady = true;
                 this.updateEditorContent(this.state.dataDictionary);
+            } else {
+                this.fastMessageSystem.postMessage({
+                    type: MessageSystemType.navigation,
+                    action: MessageSystemNavigationTypeAction.update,
+                    activeDictionaryId: e.data.value === "" ? "root" : e.data.value,
```

Comment:
We could just have "root" as a const outside the class, it's there in the constructor but you could just move it then refer to the const here.

Reply:
Done